src/modules/ekf2/EKF/aid_sources/airspeed/airspeed_fusion.cpp

```diff
@@ -96,7 +96,8 @@ void Ekf::controlAirDataFusion(const imuSample &imu_delayed)
 		_innov_check_fail_status.flags.reject_airspeed =
 			_aid_src_airspeed.innovation_rejected; // TODO: remove this redundant flag
 
-		const bool continuing_conditions_passing = _control_status.flags.in_air
+		const bool continuing_conditions_passing = _control_status.flags.in_air && (_control_status.flags.fixed_wing
```

Comment:
I wonder how much we really gain by allowing the fusion to start during transition. It only really matters if `EKF2_ARSP_THR` is considerably lower than `VT_ARSP_TRANS`. May be cleaner/simpler to just go back to only fuse it in FW flight.
@bresch was the original intent to also allow airspeed fusion for non-FW vehicles?